src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -184,7 +217,10 @@ public int doWork() {
 
         final List<SinglePassSamProgram> programs = new ArrayList<SinglePassSamProgram>();
         for (final ProgramInterface program : new HashSet<ProgramInterface>(programsToRun)) {
-            final SinglePassSamProgram instance = program.makeInstance(OUTPUT , INPUT, REFERENCE_SEQUENCE);
+            if (program.needsReferenceSequence() && REFERENCE_SEQUENCE==null) {
+                throw new PicardException("A program you are trying to run needs a Reference Sequence, please set REFERENCE_SEQUENCE in the command line");
```

Comment:
I was hoping to be able to but couldn't figure it out right away. Let me look into it further.